build-parent/pom.xml

```diff
@@ -101,6 +101,9 @@
         <keycloak.docker.image>quay.io/keycloak/keycloak:${keycloak.version}</keycloak.docker.image>
         <keycloak.docker.legacy.image>quay.io/keycloak/keycloak:${keycloak.version}-legacy</keycloak.docker.legacy.image>
 
+        <!-- Note: this version is also set in quarkus-bom but BOMs don't contribute to pluginManagement or properties -->
+        <kubernetes-client.version>6.2.0</kubernetes-client.version>
```

Comment:
This property should be defined in one place. If it's defined here then it should be removed from the BOM https://github.com/quarkusio/quarkus/blob/main/bom/application/pom.xml#L158, where it is currently set to 6.3.1, btw.

Reply:
Having it in one place will also make it easier for Dependabot to update things

Reply:
👍 moved to the root-level `pom.xml` to be available.

Do you want me to move also this one:
https://github.com/quarkusio/quarkus/blob/b6055a22014f62174c7a1275ce2bc884a7091c5b/build-parent/pom.xml#L117-L118
? (which is the reason of the first implementation)

Reply:
I'd say yes. Thanks for noticing, I wasn't aware of this one.

Reply:
The jacoco change can be a separate commit :)

Reply:
Here you go: https://github.com/quarkusio/quarkus/pull/29928/commits/aaddaf9dd597cfccc9f1041a2b033592e6b16804